resources/migrations/035_modify_setting_value_length.yaml

```diff
@@ -0,0 +1,12 @@
+databaseChangeLog:
+  - changeSet:
+      id: 35
+      author: agilliland
+      changes:
+        - modifyDataType:
+            tableName: setting
+            columnName: value
+            newDataType: TEXT
```

Comment:
TEXTs need to go through `jdbc-clob->str` when they come out of the DB looks like your tests were failing because of that